docs(elements): fixed command that adds the package @angular/elements

guiseek opened the pull request:
I was using schematics with the --name parameter instead of the --project, I did both ways before sending and my suspicion about outdated documentation was confirmed.

gkalpak reviewed (approved):
LGTM :+1:
Thx, @guiseek!

gkalpak commented:
BTW, the CI failure is not caused by this PR and will be fixed by #37679.
@guiseek, can you rebase your PR on latest master once #37679 has been merged?

IgorMinar reviewed (changes requested):
@guillaumebarranco thanks for the contribution. Before we can process it, we'll need your help to update this PR. Can you please rebase this PR and remove all of the unrelated changes? thanks!

guiseek committed:
docs(elements): fixed command that adds the package @angular/elements

I was using schematics with the `--name` parameter instead of the `--project`, I did both ways before sending and my suspicion about outdated documentation was confirmed

IgorMinar reviewed (approved):
LGTM now. Thank you @guiseek for the rebase and cleanup!

guiseek commented:
> LGTM now. Thank you @guiseek for the rebase and cleanup!



> LGTM now. Thank you @guiseek for the rebase and cleanup!

I appreciate it, thank you.
